bus-mapping/src/circuit_input_builder/access.rs

```diff
@@ -4,7 +4,7 @@ use ethers_core::utils::get_contract_address;
 use std::collections::{hash_map::Entry, HashMap, HashSet};
 
 /// State and Code Access with "keys/index" used in the access operation.
-#[derive(Debug, PartialEq)]
+#[derive(Debug, PartialEq, Eq)]
```

Comment:
Is `Eq` needed specifically?? If not, let's save space and compilation time and not derive traits unless they're required.

Reply:
make sense